tests/legacy-cli/e2e/tests/build/prod-build.ts

```diff
@@ -1,30 +1,37 @@
-import {join} from 'path';
-import {readdirSync} from 'fs';
-import {expectFileToExist, expectFileToMatch} from '../../utils/fs';
-import {ng} from '../../utils/process';
-import {expectGitToBeClean} from '../../utils/git';
+import { readdirSync } from 'fs';
+import { join } from 'path';
+import { getGlobalVariable } from '../../utils/env';
+import { expectFileToExist, expectFileToMatch } from '../../utils/fs';
+import { expectGitToBeClean } from '../../utils/git';
+import { ng } from '../../utils/process';
 
 
-export default function() {
+export default async function () {
```

Comment:
Can you also add a basic size check here please? Allowing something like the current size plus or minus 10%.